src/dataframe/dataframe.jl

```diff
@@ -120,11 +121,23 @@ function DataFrame(; kwargs...)
     end
 end
 
-function DataFrame(columns::AbstractVector,
-                   cnames::AbstractVector{Symbol} = gennames(length(columns)))
+function DataFrame(columns::AbstractVector)
```

Comment:
That function should be moved to deprecated.jl.

Reply:
moved